OracleTuxedo/core/samples/ws_ssl_svr/ws_svr_runme.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+# 
+# Shell script to build and run ws_svr.  It assume TUXDIR has been set, or that 
+# Tuxedo has been installed to: ~/tuxHome/tuxedo12.1.3.0.0   If not, invoke 
+# this script with a single argument indicating the location of TUXDIR.
+#
+# Usage: source ws_svr_runme.sh
+#
+mkdir -p /u01/oracle/user_projects
+cd /u01/oracle/user_projects || return 1
+rm -rf ws_svr
+mkdir ws_svr
+cd ws_svr || return 1
+
+# Create environment setup script setenv.sh
+HOSTNAME=$(uname -n)
+APPDIR=$(pwd)
+export HOSTNAME APPDIR
+export WS_PORT=9055
+export SSL_PORT=9060
+#export TUX_LOADBAL_IP="138.3.79.82"
```

Comment:
If you don't need this variable, remove it.

Reply:
Added the following comment:  

`# uncomment and use this if behind NAT and this is not set as env var`

Reply:
Add a note to the `README.md` that users must include `-e TUX_LOADBAL_IP` if NAT support is required and remove the commented out line from the script. Having said that, wouldn't this _always_ be required in a container using default NAT networking? If so, why isn't this set automatically by your script?

Reply:
Will remove the commented out line from the script (and will actually rename it to TUX_EXTERNAL_IP everywhere else).